content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -100,57 +116,71 @@ On your validator:
+This configuration involves connecting your validator to one or more public hubs that connect to the network. Successful public hubs embody the following traits:
 
-- **Issuing timely votes**
+- Good bandwidth.
+- Connections with a lot of reliable peers.
+- Ability to relay messages reliably.
 
-    A good validator's votes arrive quickly and not after a consensus round has already finished. To keep your votes timely, make sure your validator meets the recommended [system requirements](system-requirements.html), which include a fast internet connection.
+The benefit of connecting to public hubs is easy access to a lot of safe and reliable connections to the network. These connections help keep your validator healthy.
 
-- **Identified**
+To connect your validator to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file.
 
-    A good validator has a clearly identified owner. Providing [domain verification](#provide-domain-verification) is a good start. Ideally, XRP Ledger network UNLs include validators operated by different owners in multiple legal jurisdictions and geographic areas. This reduces the chance that any localized events could interfere with the impartial operations of trusted validators.
+1. Include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `zaphod.alloy.ee 51235`, are public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
 
-It's worth noting that running an untrusted validator can also be helpful to the overall health of the network. Untrusted validators help set the standard that trusted validators are measured against. For example, if a trusted validator is disagreeing with a lot of untrusted validators, that might indicate a problem.
+        [ips_fixed]
+        r.ripple.com 51235
+        zaphod.alloy.ee 51235
 
-Ripple (the company) publishes a [validator list](https://github.com/ripple/rippled/blob/develop/cfg/validators-example.txt) with a set of recommended validators. Ripple strongly recommends using exactly this list for production servers.
+    You can include the IP addresses of other `rippled` servers here, but _**only**_ if you can expect them to:
+
+      - Relay messages more or less unconditionally without censoring.
+      - Not go offline.
+      - Not DDoS you.
+      - Not crash you.
+      - Not publish your IP address to strangers.
+
+2. Include the following `[peer_private]` stanza and set it to `1`. Enabling this setting instructs your validator’s peers to not broadcast your validator’s IP address. This setting also instructs your validator to connect to only the peers configured in your `[ips_fixed]` stanza. This ensures that your validator connects to and shares its IP with only peer `rippled` servers you know and trust.
```

Comment:
super nit: "to not broadcast" → "not to broadcast" because split infinitives are a tool to be used boldly, not casually. ;)